src/graphic/Fast3D/gfx_sdl2.cpp

```diff
@@ -341,9 +342,9 @@ static void gfx_sdl_init(const char* game_name, const char* gfx_api_name, bool s
 
         SDL_GetRendererOutputSize(renderer, &window_width, &window_height);
         window_impl.Metal = { wnd, renderer };
+        window_impl.backend = SohImGui::Backend::SDL_METAL;
     }
 
-    window_impl.backend = SohImGui::Backend::SDL;
```

Comment:
this was concerning when i first saw it, but after looking into it, unless we're on apple, we will always hit
```cpp
window_impl.backend = SohImGui::Backend::SDL_OPENGL;
```
on L335 on non-mac systems using sdl because L314 has
```cpp
if (use_opengl) {
```
and that is set up on L265
```cpp
#if defined(__APPLE__)
    bool use_opengl = strcmp(gfx_api_name, "OpenGL") == 0;
#else
    bool use_opengl = true;
#endif
```